server/historian/package.json

```diff
@@ -15,8 +15,9 @@
     "build:compile": "lerna run build:compile --stream",
     "build:genver": "lerna run build:genver --stream --parallel",
     "clean": "rimraf dist *.tsbuildinfo *.build.log",
+    "format": "lerna run format --no-sort --stream -- -- -- --color",
     "postinstall": "npm run postinstall:lerna",
-    "lint": "lerna run lint --no-sort --stream",
+    "lerna": "lerna",
```

Comment:
Looks like the "lint" script was removed in this one, rather than the "lerna" script.